analysis/src/topEventSelectionSL.cc

```diff
@@ -64,48 +71,58 @@ int topEventSelectionSL::EventSelection()
   if (std::abs(PV_z) >= 24.) return b_step;
   if (PV_npvs == 0) return b_step;
   if (PV_ndof < 4) return b_step;
+  
+  b_nvertex = PV_npvs;
 
   if (h_cutFlow) h_cutFlow->Fill(2);
 
   //Triggers
-  b_trig_m = HLT_IsoTkMu24 || HLT_IsoMu24;
-  b_trig_e = HLT_Ele27_WPTight_Gsf;
+  b_trig_m_trk    = HLT_IsoTkMu24 ? 1 : 0;
+  b_trig_m_global = HLT_IsoMu24 ? 1 : 0;
 
-  // TODO Check trigger requirements (TTbarXSecSynchronization page doesn't have yet)
+  b_trig_m = HLT_IsoTkMu24 || HLT_IsoMu24 ? 1 : 0;
```

Comment:
Put () around the condition. Can't you use a bool for this? In which case you don't need the "?:" operator

Reply:
OK Thanks!